tools/plugins/com.liferay.ide.server.core/META-INF/MANIFEST.MF

```diff
@@ -28,7 +28,8 @@ Require-Bundle: org.eclipse.wst.server.core;visibility:=reexport,
  org.apache.commons.lang,
  org.apache.commons.io,
  org.eclipse.core.commands,
- org.eclipse.wst.validation
+ org.eclipse.wst.validation,
+ org.apache.commons.collections
```

Comment:
Don't add commons.collections as dependency, we don't need it here.

Reply:
ok